index.html

```diff
@@ -643,24 +643,25 @@ <h2>Security</h2>
           </ul>
         </div>
         <p><span class="rfc2119-assertion" id="common-constraints-security-2">
-            Conformant Consumers MUST support all of these security schemes.</span>
+            Conformant Consumers MUST support at least all of these security schemes.</span>
         </p>
-	    
+
         <p><span class="rfc2119-assertion" id="common-constraints-security-3">
-            A Thing MAY implement multiple security schemes.</span>
+            A Thing MAY implement multiple security schemes</span>,
+            <span class="rfc2119-assertion" id="common-constraints-security-4"> at least one of the above MUST be supported.</span>
```

Comment:
I suggest splitting this into two sentences. Once these assertions get extracted out into implementation reports the latter is not going to make any sense.

Reply:
I agree in principle to improve the language, however splitting into two sentences does not solve the problem. The 2nd sentence should contain the members of the list above.